CMakeLists.txt

```diff
@@ -792,3 +794,13 @@ if (ENABLE_DISTRIBUTED)
                         ${GFLAGS_LIB_NAME}
                         ${GRPCPLUSPLUS_LIBRARIES})
 endif(ENABLE_DISTRIBUTED)
+
+if(ENABLE_SINGLE_NODE_SERVER)
+  add_executable (quickstep_client cli/QuickstepClient.cpp)
+  target_link_libraries(quickstep_client
+            ${GFLAGS_LIB_NAME}
+            glog
+            ${GRPCPLUSPLUS_LIBRARIES}
+            ${PROTOBUF3_LIBRARIES}
+            quickstep_cli_SingleNodeServer_proto)
```

Comment:
All linking libraries should be aligned with `(`, and sort in the alphabet order.

```
             glog
             quickstep_cli_SingleNodeServer_proto
             ${GFLAGS_LIB_NAME}
             ${GRPCPLUSPLUS_LIBRARIES})
```